<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CasseNoisetteClass</class>
 <widget class="QMainWindow" name="CasseNoisetteClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CasseNoisette</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Fichier contenant les mots de passes cryptés : </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="pwdFileSelectTxt"/>
         </item>
         <item>
          <widget class="QPushButton" name="pwdFileSelectBtn">
           <property name="text">
            <string>Parcourir...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="SeperatorLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Séparateur : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtPwdsSeperator">
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Fonction de hachage : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="hashFunctionsComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>MD5</string>
           </property>
           <item>
            <property name="text">
             <string>MD5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SHA-1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SHA-256</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SHA-512</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="GuessHashCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Deviner?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblEnginDesc">
      <property name="text">
       <string>Engin de cassage :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="bruteForceTab">
       <attribute name="title">
        <string>Force Brute</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="CharsetSelector" name="charsetSelectorWidget" native="true"/>
        </item>
        <item>
         <widget class="QGroupBox" name="crackingRulesGroupBox">
          <property name="title">
           <string>Règles</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QSpinBox" name="spinMaxPwdLenght">
             <property name="wrapping">
              <bool>false</bool>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="accelerated">
              <bool>false</bool>
             </property>
             <property name="showGroupSeparator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Longueur Max : </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dictionaryTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Dictionnaire</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Liste de mots</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Fichier contenant la liste des mots (Dictionnaire) :</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="dictFileSelectTxt"/>
             </item>
             <item>
              <widget class="QPushButton" name="dictFileSelectBtn">
               <property name="text">
                <string>Parcourir...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rainbowTableTab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Table arc-en-ciel</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Table arc-en-ciel</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Fichier contenant la table arc-en-ciel (Rainbow Table) :</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>Parcourir...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="startCrackBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">padding:5;</string>
      </property>
      <property name="text">
       <string>Démarrer le cassage</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_propos">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="generateDictionaryBtn"/>
    <addaction name="separator"/>
    <addaction name="aboutBtn"/>
   </widget>
   <addaction name="menu_propos"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aboutBtn">
   <property name="text">
    <string>À propos</string>
   </property>
  </action>
  <action name="generateDictionaryBtn">
   <property name="text">
    <string>Générer un dictionnaire</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CharsetSelector</class>
   <extends>QWidget</extends>
   <header>CharsetSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cassenoisette.qrc"/>
 </resources>
 <connections/>
</ui>
